packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -738,17 +738,15 @@ function handleCreateDeployError(output, error, localConfig) {
     return 1;
   }
   if (error instanceof SchemaValidationFailed) {
-    const { message, params, keyword, dataPath } = error.meta;
-
+    const ajvError = error.meta;
+    const fileName = localConfig[fileNameSymbol];
+    const humanError = humanizeAjvError(ajvError, fileName);
+    output.prettyError(humanError);
+    return 1;
+    /*
```

Comment:
How come this is being commented out / not deleted?

Reply:
Thanks I forgot to implement this part since there was no test. I just added a "suggestions" feature which should cover this use case.